lib/new_relic/control/security_interface.rb

```diff
@@ -0,0 +1,40 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require 'singleton'
+
+module NewRelic
+  class Control
+    class SecurityInterface
+      include Singleton
+
+      attr_accessor :wait
+
+      def agent_started?
+        (@agent_started ||= false) == true
+      end
+
+      def waiting?
+        (@wait ||= false) == true
+      end
+
+      def init_agent
+        return if agent_started? || waiting?
+
+        if Agent.config[:'security.agent.enabled']
```

Comment:
Don't we need both `security.agent.enabled` and `security.enabled` to be true to init the security agent?

Reply:
Yes. The APM agent checks for one to guard the `require` and the security agent checks for the other one to guard the actually starting of the security agent and running its event loop.